proxy/InkAPI.cc

```diff
@@ -9586,3 +9591,35 @@ TSRegisterProtocolTag(const char *tag)
 {
   return nullptr;
 }
+
+tsapi char *
+TSHttpTxnGetClientRequestBody(TSHttpTxn txnp, int *len)
```

Comment:
This should be TSHttpTxnClientReqBodyGet() . 1) Get() at the end, and 2) bcall pointed out that we have a TSHttpTxnClientReqGet().

Reply:
Thanks, zwoop.